docs/src/makie.md

```diff
@@ -0,0 +1,66 @@
+# Makie.jl plots
+
+Compared to Gadfly.jl and StatsPlots.jl, Makie.jl is the most flexible plotting library that you can use.
+On this page is an example function for plotting with Makie.jl which you can use directly or further tweak to your needs.
+
+```@example makie
+using CairoMakie
+using DataFrames
+using MCMCChains
+
+chns = Chains(randn(300, 5, 3), [:A, :B, :C, :D, :E])
+```
+
+```@example makie
+function plot_chains(chns; density_func=density!)
+    params = names(chns, :parameters)
+
+    df = DataFrame(chns)
```

Comment:
Can we use the `Chains` object directly instead of converting to a `DataFrame`.
I think this is possible because `MCMCChains.jl` is a `Tables.jl` API-friendly interface.

Reply:
I know. Feel encouraged to show such an implementation

Reply:
Solved via directly pulling the right elements from the axisarrays. Those arrays store different chains in different matrix columns... :sweat_smile: